### PR TITLE
Don't run weird pip install test on CentOS 5

### PR DESCRIPTION
**Pull request by rallytime**
The egg needed to run more complex tests fails on CentOS 5 with an upstream bug. Let's skip it there.

### CODE REVIEW

File: tests/integration/states/pip.py [hunk 1]
```diff
@@ -64,6 +64,16 @@ def test_pip_installed_errors(self):
                 shutil.rmtree(venv_dir)
```
Comment (s0undt3ch) [inline tests/integration/states/pip.py]: I think we have a with grains decorator I'm salt-testing, not that this approach is wrong...

Comment (s0undt3ch) [inline tests/integration/states/pip.py]: In!

Comment (rallytime) [inline tests/integration/states/pip.py]: Oh we do? That would be swell. I briefly looked around a didn't see one, but I can reassess.

Comment (s0undt3ch) [inline tests/integration/states/pip.py]: https://github.com/saltstack/salt-testing/blob/develop/salttesting/helpers.py#L939-L954

Comment (s0undt3ch) [inline tests/integration/states/pip.py]: In use [here](https://github.com/saltstack/salt/blob/develop/tests/integration/modules/useradd.py#L41).

**Pull request merged**